x-pack/plugins/endpoint/public/applications/endpoint/view/policy/policy_list.tsx

```diff
@@ -160,7 +160,7 @@ export const PolicyList = React.memo(() => {
       }
     >
       <EuiBasicTable
-        items={policyItems}
+        items={useMemo(() => [...policyItems], [policyItems])}
```

Comment:
`EuiBasicTable` doesn't accept readonly arrays. We could probably get that changed in EUI, as it probably doesn't actually mutate `items`